index.html

```diff
@@ -3,39 +3,43 @@
  <head>
    <title>The Gallery of Horns</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-   <link rel="stylesheet" href="layouts.css">
+   
+   <script id="creature-template" type="text/x-handlebars-template">
+    <section class="card" data-horns="{{horns}}" data-keyword="{{keyword}}">
+      <h2>{{title}}</h2>
+      <img
+        src="{{image_url}}"
+        alt="{{title}}">
+      <p>{{description}}</p>
+    </section>
+  </script>
+
+  <script id="creature-option-template" type="text/x-handlebars-template">
+    <option value="{{keyword}}">{{keyword}}</option>
+  </script>
+
+   <link rel="stylesheet" href="../css/layouts.css">
  </head>
  <body>
    <header>
-        <h1>The Gallery of Horns</h1>
-        <select id="horned_animals">
-          <option value="default">Filter by Keyword</option>
-          <option value="narwhal">Narwhal</option>
-          <option value="rhino">Rhino</option>
-          <option value="unicorn">Unicorn</option>
-          <option value="unilego">Unilego</option>
-          <option value="triceratops">Triceratops</option>
-          <option value="markhor">Markhor</option>
-          <option value="mouflon">Mouflon</option>
-          <option value="addax">Addax</option>
-          <option value="chameleon">Chameleon</option>
-          <option value="lizard">Lizard</option>
-          <option value="dragon">Dragon</option>
+    <h1>The Gallery of Horns</h1>
+      <select>
+        <option value="default">Filter by Keyword</option>

```

Comment:
If the point of line 29 is to indicate to the user what this selection box does, you can do it differently. A very common pattern is to make this first option tag (on line 27) give the instructions to the user to indicate what this select element does.